### DISCUSSION TITLE
Debugging failing Azure Docker instance

### DISCUSSION THREAD

**Discussion by marcsroberts** [getfider/fider#997] [Q&A]
Good morning!

I've been chatting to Guilherme by email, but thought it may be easier to continue here. We've been using an Azure hosted Docker instance of Fider since March 2021. We followed the steps at https://getfider.com/docs/hosting-on-azure and other than using our own domain name haven't changed anything from defaults.

Things were going fine until the early morning of the 18th of August 2021 when it appears an update to 19 happened, now the site is failing to spin up. This is the most detail I've been able to find about the time of death and subsequent looping log entries:
![Screenshot 2021-08-20 at 10 21 19](https://user-images.githubusercontent.com/49167833/130211723-41642aef-d648-4a40-86e3-e2a661185252.png)
for clarity:
![Screenshot 2021-08-20 at 10 35 50](https://user-images.githubusercontent.com/49167833/130213478-098e74ae-fffc-4f14-9f47-840357a04981.png)

I've tried creating a new clean Web App in Azure, but the end result is the same. Guilherme suggested we try running v18.1, but I'm not sure how to do that in our Azure instance.

I'm wondering if anyone is able to give me any pointers to debug the issue? I'm new to Docker...

Thank you!

Marc

PS: Docker logs, not much in them:
<img width="926" alt="Screenshot 2021-08-20 at 10 26 50" src="https://user-images.githubusercontent.com/49167833/130212376-3741a1a9-c7ad-4f4d-b2ad-31693862d361.png">
<img width="1731" alt="Screenshot 2021-08-20 at 10 28 29" src="https://user-images.githubusercontent.com/49167833/130212736-0a771ba2-fc25-4ce8-8c54-5e4cc8ed47f1.png">

*Update*
We've rolled it back to 0.18.1 and it's working again, so all pointing to a 19 migration issue with our data?
![Screenshot 2021-08-20 at 13 24 13](https://user-images.githubusercontent.com/49167833/130232732-37a0227b-74dd-45e3-8bc2-d16442678f6c.png)

**Comment by goenning** [comment]
Thanks for the detailed info. It seems like the container is running but Azure is unable to ping it, so it crashes after some time.

Can you set LOG_LEVEL to DEBUG (env variables/appsettings) and see if that gives us more info?

**Comment by goenning** [comment]
@marcsroberts  this should be fixed on `getfider/fider:main` image, can you try again and let me know? Thanks

**Reply by marcsroberts** [reply]
Sure thing, I’ll see if I can give it a try or ask my colleague to early next week and let you know!

If I can do it tomorrow, I’ll let you know.
